tests/jballerina-integration-test/src/test/resources/configurables/configStructuredTypesProject/configStructuredTypes/main.bal

```diff
@@ -275,25 +277,25 @@ public function testMaps() {
     test:assertEquals(readonlyManagerMap.toString(), "{\"m1\":{\"name\":\"manu\",\"id\":333},\"m2\":{\"name\":" +
     "\"riyafa\",\"id\":444}}");
 
-    testMapIterator(user, 3);
-    testMapIterator(numbers, 4);
-    testMapIterator(fractions, 3);
-    testMapIterator(bits, 2);
-    testMapIterator(numberSet, 3);
-    testMapIterator(stringSet, 3);
-    testMapIterator(engineerMap, 3);
-    testMapIterator(lecturerMap, 2);
-    testMapIterator(readonlyEngineerMap, 3);
-    testMapIterator(readonlyLecturerMap, 2);
-    testMapIterator(subjects, 3);
-    testMapIterator(departments, 3);
-    testMapIterator(intMap, 3);
-    testMapIterator(libIntMap, 3);
-    testMapIterator(managerMap, 2);
-    testMapIterator(readonlyManagerMap, 2);
+    mod3:testMapIterator(user, 3);
+    mod3:testMapIterator(numbers, 4);
+    mod3:testMapIterator(fractions, 3);
+    mod3:testMapIterator(bits, 2);
+    mod3:testMapIterator(numberSet, 3);
+    mod3:testMapIterator(stringSet, 3);
+    mod3:testMapIterator(engineerMap, 3);
+    mod3:testMapIterator(lecturerMap, 2);
+    mod3:testMapIterator(readonlyEngineerMap, 3);
+    mod3:testMapIterator(readonlyLecturerMap, 2);
+    mod3:testMapIterator(subjects, 3);
+    mod3:testMapIterator(departments, 3);
+    mod3:testMapIterator(intMap, 3);
+    mod3:testMapIterator(libIntMap, 3);
+    mod3:testMapIterator(managerMap, 2);
+    mod3:testMapIterator(readonlyManagerMap, 2);
     // These lines should be enabled after fixing #30566
-    // testMapIterator(studentMap, 3);
-    // testMapIterator(readonlyStudentMap, 3);
+    // mod3:testMapIterator(studentMap, 3);
+    // mod3:testMapIterator(readonlyStudentMap, 3);
```

Comment:
Shall we give  the modules proper names so that they are more representative as discussed offline?